FUSION_Main.py

```diff
@@ -4840,7 +4840,7 @@ def start_cluster_markers(self,butt_click):
             labels = np.unique(self.feature_data['label'].tolist())
             slide_ids = np.unique([i['Slide_Id'] for i in self.feature_data['Hidden'].tolist()])
 
-            return [marker_logs_children],[disable_button], json.dumps({'plugin_used': {'clustermarkers_fusion': { 'features': self.feature_data, 'label': labels, 'slide_ids': slide_ids } } })
+            return [marker_logs_children],[disable_button], json.dumps({'plugin_used': 'clustermarkers_fusion', 'features': self.feature_data, 'label': labels, 'slide_ids': slide_ids })
```

Comment:
Let's just record the names of the features instead of all the features and their values. This can be switched to:
`'features': self.feature_data.columns.tolist()`